nodejs/awsx/apigateway/api.ts

```diff
@@ -593,13 +669,20 @@ function addStaticRouteToSwaggerSpec(
         }, parentOpts);
     }
 
-    function processFile(route: StaticRoute) {
+    function processFile(route: StaticRoute, requiredParams?: reqvalidation.Parameter[]) {
```

Comment:
in general, for internal APIs, it's good to not make the params optional (since people might forget them).  instead, do this: `requiredParam: Parameter[] | undefined`.  this makes hte parameter something the caller has to pass, but it's acceptable for them to pass an undefined value.

Reply:
Looks like that was left over / I did not it anymore. But thats a great tip thanks.